spec/acceptance/stub_spec.rb

```diff
@@ -58,3 +58,44 @@
     expect(subject.user).not_to be_new_record
   end
 end
+
+describe "a stubbed instance with no primary key" do
+  it "builds a stubbed instance" do
+    using_model_without_pk do
```

Comment:
this definitely addresses the case without a primary key, I think my question would be: Do we even wanna support cases where the primary key is not the id field?